rclpy/test/test_waitable.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, node):
         period_nanoseconds = 10000
         with self._clock.handle, node.context.handle:
             self.timer = _rclpy.Timer(
-                self._clock.handle, node.context.handle, period_nanoseconds)
+                self._clock.handle, node.context.handle, period_nanoseconds, True)
```

Comment:
Whoul'd not this `True` value be added to the parameters list of this `__init__` method? Maybe with default `True` value?

Reply:
I don't really see a need to.  This is for a specific test, and this test wants the timer to be automatically started, so there is no reason in my opinion to make it configurable.